### PR TITLE
Adding spacy-pkuseg package

### PR DESCRIPTION
**Pull request by adrianeboyd**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ines** [comment]
Confirming that I'm willing to be listed as a maintainer 🙂

File: recipes/spacy-pkuseg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - {{ compiler('cxx') }}
+    - cython
+    - numpy >=1.16.0
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+    - cython
+    - numpy >=1.16.0
```
Comment (scopatz) [inline recipes/spacy-pkuseg/meta.yaml:31]: If you are linking against numpy: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Comment (adrianeboyd) [reply recipes/spacy-pkuseg/meta.yaml]: Thanks for the feedback, I hope the update fixed this correctly?

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**